displays/cavity_display/frontend/fault_count_display.py

```diff
@@ -119,8 +120,9 @@ def get_data(self):
 
```

Comment:
Oh, apparently I can't leave a comment on an unchanged line?! Oh well. I'm referring to lines 116-119 above.

It looks like we can break the GUI if we check "Hide POT faults" but there were actually no POT faults in the data dictionary (terminal error is `KeyError: 'POT'`)

Reply:
Looks like we just need to check to see if the "POT" key exists. So maybe changing the lines to 
`if self.hide_pot_checkbox.isChecked() and "POT" in data:`
`    data.pop("POT")`